crates/solver/src/driver.rs

```diff
@@ -656,4 +669,52 @@ mod tests {
         );
         assert!(!is_only_selling_trusted_tokens(&settlement, &token_list));
     }
+
+    #[test]
+    #[ignore]
+    fn print_settlements_() {
```

Comment:
nit: I guess it would be nice name the test `print_settlements` (without _).
This could be done by calling the actual `print_settlements` function with `super::print_settlements`.